Apps/AccountDataAccess/src/Patient/PatientRepository.cs

```diff
@@ -191,10 +192,18 @@ private async Task<PatientQueryResult> Handle(PatientDetailsQuery query, Cancell
             throw new InvalidOperationException("Must specify either Hdid or Phn to query patient details");
         }
 
-        private async Task<PatientModel> GetPatientIdentityAsync(string hdid)
+        private async Task<PatientModel?> GetPatientIdentityAsync(string hdid)
         {
-            PatientIdentity result = await this.patientIdentityApi.GetPatientIdentityAsync(hdid).ConfigureAwait(true);
-            return this.mapper.Map<PatientModel>(result);
+            try
+            {
+                PatientIdentity result = await this.patientIdentityApi.GetPatientIdentityAsync(hdid).ConfigureAwait(true);
+                return this.mapper.Map<PatientModel>(result);
+            }
+            catch (ApiException e) when (e.StatusCode == HttpStatusCode.NotFound)
+            {
+                this.logger.LogError("PHSA could not find patient identity for {Hdid}", hdid);
```

Comment:
not sure if this is considered an error - it is a legitimate response

Reply:
I changed the level to Information

Reply:
The service level logging was not clear so added this to ensure where the not found was coming from.